source/features/repo-avatars.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import elementReady from 'element-ready';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+
+async function init(): Promise<void> {
+	// Icon for public but not template/fork/etc. repos
+	const icon = await elementReady('#repository-container-header .octicon-repo');
+	if (!icon) {
+		return;
+	}
+
+	const author = select('#repository-container-header [rel=author]')!;
+	const username = author.textContent!;
+	const alt = `@${username}`;
+	const fallbackSrc = (pageDetect.isEnterprise()
+		? `/${username}.png`
+		: `https://avatars.githubusercontent.com/${username}`) + '?size=48';
+	const src = select(`img[alt="${alt}"]`)?.src ?? fallbackSrc;
```

Comment:
We have a nearly-identical piece of code in `reaction-avatars`, which also takes display density into account.

Reply:
This is indeed adapted from `reations-avatars`, just haven't thought of making it into a function.